src/NotificationStack.vala

```diff
@@ -95,7 +99,6 @@ public class Gala.NotificationStack : Object {
         var scale = Utils.get_ui_scaling_factor ();
         stack_width = (WIDTH + MARGIN) * scale;
 
-        stack_x = area.x + area.width - stack_width;
```

Comment:
More of a question, can't the X coordinate of the notification be calculated here?

Reply:
My thinking was that it's possible (but unlikely) for different notifications to be different widths, so I wanted to account for that. `update_stack_allocation` is only called when the size/layout of the monitors is changed.